src/verilog/testbench.cpp

```diff
@@ -0,0 +1,33 @@
+#include "VTop.h"
```

Comment:
Why this file added?

Reply:
When generating verilog from elaboration a testbench file is needed to run the verilated verilog. This file is just a simple testbench that clocks the pure verilated design.